app/furniture/marketplace/orders/_order.html.erb

```diff
@@ -26,4 +26,18 @@
   <hr class="col-span-6 mb-2" />
   <div class="col-span-4">Total</div>
   <div class="col-span-2 text-right"><%= humanized_money_with_symbol(order.price_total) %></div>
+
+  <%- if policy(order.events).index? %>
+    <hr class="col-span-6 my-4" />
+    <div class="col-span-6 text-xs">
+
+      <h3 class="mb-2 text-sm font-bold">Events</h3>
```

Comment:
Is "Events" a friendly term? What about "Timeline"?

Reply:
I like that!